frontend/javascripts/oxalis/view/layouting/golden_layout_adapter.js

```diff
@@ -103,6 +104,32 @@ export class GoldenLayoutAdapter extends React.PureComponent<Props<*>, *> {
     if (onLayoutChange != null && this.gl.isInitialised) {
       onLayoutChange(this.gl.toConfig(), this.props.activeLayoutName);
     }
+    // If the maximized item changed adjust css classes to not show hidden gl items.
+    if (this.gl && this.maximizedItem !== !this.gl._maximisedItem) {
```

Comment:
I think the entire if could be moved after line 105, since it should only be executed if `this.gl.isInitialised` is true, right? Maybe this even makes another condition in the if-head obsolete?

Reply:
I moved it after line 105 as you suggested. But it does not make the whole if statement obsolete because the content of the if shall only be executed when the maximized component really changed to save performance. Otherwise, this would be even executed when just changing the layout without any maximized component before and after.